tools/cldr-apps/js/src/esm/cldrStatus.js

```diff
@@ -125,6 +126,7 @@ function setCurrentId(id) {
     id = id.toString();
     currentId = idIsAllowed(id) ? id : "";
   }
+  cldrRef.xpath.value = currentId;
```

Comment:
a setter would obviate including Vue implementation concerns like ".value"

Reply:
what if "id" is not an xpath at all? sometimes it identifies a user or a forum post

Reply:
I thought this particular currentId was always an xpath.

Reply:
unfortunately not